### PR TITLE
Remove the code that calls the origin migrator

### PR DESCRIPTION
**Pull request by dbkr**
This will have done its job now, everyone's had long enough to
install a newer version of Riot and migrate to the new origin.

Laves the code on the backend that handles it for the time being,
as per comment.

### CODE REVIEW

File: electron_app/src/electron-main.js [hunk 1]
```diff
@@ -232,6 +232,10 @@ ipcMain.on('ipcCall', async function(ev, payload) {
             }
             break;
         case 'origin_migrate':
+            // NB. The code to call this has now been removed so this
+            // should never be called. This code itself has been removed
+            // with the migration to the riot-desktop repo, with the
+            // expectation that this will become the new home for this code.
```
Comment (jryans) [inline electron_app/src/electron-main.js]: Hmm, even after reading this comment, I am still puzzled as to why we're leaving this `origin_migrate` message handler in place now that nothing calls it... Will we remove it at some future time?

Comment (dbkr) [reply electron_app/src/electron-main.js]: Sorry, have tried to explain better! We could also just remove the code in both places, it's not actually very invasive.

Comment (jryans) [reply electron_app/src/electron-main.js]: Hmm, I guess it just feels strange to remove it half-way like this... What if we end up abandoning the separate repo experiment for some reason? Then we'll be left with this half-way removed thing that we're likely to forget about.

I'd suggest removing the message handler and the module over here as well.

Comment (dbkr) [reply electron_app/src/electron-main.js]: Fair enough - removed here too

**Pull request merged**